### DISCUSSION TITLE
Primera semana de diseño web y he entendido...

### DISCUSSION THREAD

**Discussion by escuelaDeCodigoMargaritaMaza** [escuelaDeCodigoMargaritaMaza/escuela_de_codigo#4] [General]
En esta semana que te llevas?

**Comment by MaribelZarate** [comment]
He entendido la diferencia entre página web sitio web y he aprendido a hacer un boceto y el buyer persona los requerimientos para la elaboración de la página

**Comment by OmarJoasAD** [comment]
Me llevo el tema de que es un Algoritmo, su importancia a la hora de programar, los tipos de datos, también su importancia a la hora de programar, los operadores aritméticos, relacionales y lógicos, como se manejan las computadoras y así, todo lo que hemos aprendido ésta semana. 👍🏻👍🏻👏🏻

**Comment by GerardoLopezPadilla** [comment]
Yo en lo personal aprendí de manera básica como es que funciona todo este mundo del diseño web, así como las estructuras y procedimientos, reitero, básicos, también aprendí el como utilizar herramientas como lo son los wireframes, así como el procedimiento recomendado para la elaboración de un buen diseño.

**Comment by Elizagtierrez** [comment]
Hola, esta primera semana aprendí sobre el internet como una red de redes que atraviesan mares y océanos, así  como nació el diseño web como lo conocemos ahora, lo que es el front end que es lo que vemos de una página web; y el backend procesa la información que entra y permite las interacciones, así también se le llama full-stack a quien maneja las dos anteriores. También aprendí que para hacer una página web utilizamos las tecnologías de HTML y CSS, así como los Frameworks que son una variante del JS que nos ayudan para realizar mejor las páginas web. Aunque esta semana nos enfocamos en los primeros pasos del llamado ciclo de vida del software que básicamente son los pasos a seguir para desarrollar páginas web, el primer lugar están los requerimientos donde necesitamos plantear las necesidades primeras de la página, buscar a nuestro buyer person y nuestro target para que sea visitada la página, buscar modelos de páginas web para basarnos en el diseño y hacer otra, los colores, botones, etc.  Posteriormente se hace el diseño con la maquetación del proyecto en alguna herramienta como balsamic para poder corregir o modificar el diseño, luego se hace en figma. Para después pasar al Desarrollo, luego a la Verificación y posteriormente al mantenimiento y evaluación, para ello se utilizan diferentes herramientas, para la evaluación hay una en google donde te puede dar el posicionamiento de la página en  google, entre otros datos. Es importante siempre documentar este proceso.

**Comment by IsaiPrez** [comment]
Esta semana aprendi la importancia y el impacto de la tecnología hoy en día, es por lo anterior que me motiva a continuar con este curso puesto que a pesar de que un detalle desapercibido por nosotros, este puede ser un gran diferenciador a lo que queremos transmitir
